idl/github.com/uber/cadence/history.thrift

```diff
@@ -74,6 +74,8 @@ struct GetMutableStateResponse {
   90: optional string clientImpl
   100: optional bool isWorkflowRunning
   110: optional i32 stickyTaskListScheduleToStartTimeout
+  120: optional i32 eventStoreVersion
+  130: optional binary branchToken
```

Comment:
What about treeID?

Reply:
branchToken is enough for querying historyV2 because it includes treeID internally